File: gin/config/socket.go
package config

type SocketType struct {
	Type string
	URL  string
	Host string
	Port string
}

/* Socket配置 */
func Socket() *SocketType {
	cfg := &SocketType{}
	cfg.Type = "ws"        //类型
	cfg.URL = "/websocket" //地址
	cfg.Host = "localhost" //主机
	cfg.Port = "9031"      //端口
	return cfg
}

File: python/task/socket.py
from task.base import Base
from config.socket import Socket as cfg
from service.socket import Socket as SocketServer

import asyncio
import websockets

# Socket
class Socket(Base):

  # 启动
  def Start(self):
    server = websockets.serve(SocketServer().Open, cfg.Host, cfg.Port)
    asyncio.get_event_loop().run_until_complete(server)
    asyncio.get_event_loop().run_forever()